x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -168,6 +178,48 @@ export function AlertDetails() {
   const AlertDetailsAppSection = ruleTypeModel ? ruleTypeModel.alertDetailsAppSection : null;
   const timeZone = getTimeZone(uiSettings);
 
+  const OVERVIEW_TAB_ID = 'overview';
+  const METADATA_TAB_ID = 'metadata';
+
+  const overviewTab = AlertDetailsAppSection && rule && alertDetail?.formatted && (
+    <>
+      <EuiSpacer size="l" />
+      <AlertSummary alertSummaryFields={summaryFields} />
```

Comment:
I think we would like to show `AlertSummary` even if we don't have `AlertDetailsAppSection`, wdyt?

Reply:
updated in 1c25061873737610b24e7a141ec555517971ed52